tests/languages/arturo!+bash/sh-string_feature.test

```diff
@@ -0,0 +1,18 @@
+{!sh
+    ls -la test/*x
+}
+
+----------------------------------------------------
+
+[
+    ["sh-string", [
+            ["string", "{!sh"],
+            ["embedded", "\r\n    ls -la test/*x\r\n"],
```

Comment:
Bash is supposed to highlight `ls`. This is because our Bash grammar doesn't have an `sh` alias (as I just found out), but it does have [`shell`](https://github.com/PrismJS/prism/blob/0d49553c6b8ce39ddb5c2d9ca6160ed2c263902c/components/prism-bash.js#L226), so please change the language definition to use `createLanguageString('shell')` instead.

Reply:
Done 😉